hazelcast/src/test/java/com/hazelcast/query/impl/getters/GetterFactoryTest.java

```diff
@@ -58,6 +77,15 @@ public void newFieldGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldT
     }
 
     @Test
+    public void newMethodGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldType() throws Exception {
```

Comment:
In this case I would create a class 

NewMethdoGetterFactoryTest

so that a test with name

newMethodGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldType

can be simplified to

whenExtractingFromSimpleField_thenInferTypeFromFieldType